### PR TITLE
add 'known issues' section to Changelog

### LINKED ISSUE #3381: [3.9.0] Document any known-issues in Release notes
This is space to list and discuss any known-issues that should perhaps be mentioned in the 3.9.0 Release notes.

### PR DESCRIPTION
**Pull request by brianlheim**
Closes #3381

Please review, not sure if this is the correct place for this.

### CODE REVIEW

**Comment by snappizz** [comment]
thanks for submitting! this should go into the changelog, which we then automatically convert to schelp using your script. (sorry for my tardiness in reviewing #3373, i'll get on it in a few hours)

**Comment by brianlheim** [comment]
@snappizz OK! Do you have any opinion about where in the changelog it should go?

**Comment by snappizz** [comment]
i think it's good to put it right at the top, after the blurb and before the first "General: Added" section.

**Comment by brianlheim** [comment]
Oops. Forgot they should be markdown links in the changelog. Fixing that now.

File: CHANGELOG.md [hunk 1]
```diff
@@ -19,6 +19,25 @@ scztt, simdax, smoge, smrg-lm, snappizz, telephon, thormagnusson,
...
+
+The IDE server status display turns yellow after a few seconds when opening `s.makeGui`.  This does not cause any usability issues ([#3310](https://github.com/supercollider/supercollider/issues/3310)).
+
+Only the first pages of the HTML files produced by SCDoc are printed in web browsers ([#3395](https://github.com/supercollider/supercollider/issues/3395)).
+
+The help browser does not remember the last position open in a document when navigating through history, and just jumps to the top of the file ([#3396](https://github.com/supercollider/supercollider/issues/3396)).
+
+Supernova loads plugins from "Extensions/plugins" rather than "Extensions" ([#3391](https://github.com/supercollider/supercollider/issues/3391)).
+
+Examples in the LevelIndicator help file are broken ([#3398](https://github.com/supercollider/supercollider/pulls/3398)).
```
Comment (patrickdupuis) [inline CHANGELOG.md]: This link should be .../supercollider/supercollider/**pull**/3398

Comment (patrickdupuis) [reply CHANGELOG.md]: I think the issue is that `LevelIndicator.style()` is broken, not the examples.

File: CHANGELOG.md [hunk 2]
```diff
@@ -19,6 +19,25 @@ scztt, simdax, smoge, smrg-lm, snappizz, telephon, thormagnusson,
 tiagomoraismorgado88, timsutton, vivid-synth, vividsnow, yurivict, and many
 more in the SC community who helped in ways other than participation on GitHub.
 
+Known Issues
+------------
+
```
Comment (patrickdupuis) [inline CHANGELOG.md]: We should add: 
- File.copy crashes interpreter when destination exists ([#3401](https://github.com/supercollider/supercollider/issues/3401)).

**Comment by brianlheim** [comment]
@patrickdupuis thanks for the review! I've made the changes you requested.

**Pull request merged**